Protobuf-C++.podspec

```diff
@@ -1,42 +0,0 @@
-Pod::Spec.new do |s|
```

Comment:
This file should not be deleted.
I guess we need to add this to some internal list.